public/app/plugins/datasource/loki/datasource.test.ts

```diff
@@ -1323,13 +1323,14 @@ describe('LokiDatasource', () => {
       expect(ds.getSupplementaryRequest(SupplementaryQueryType.LogsVolume, options)).toBeDefined();
     });
 
-    it('does not create provider if there is only an instant logs query', () => {
+    it('does create provider if there is only an instant logs query', () => {
```

Comment:
If instant log queries are not supported, then I don't understand these test cases.